source/backend/api/Services/IPropertyService.cs

```diff
@@ -28,8 +28,14 @@ public interface IPropertyService
 
         PropertyManagementModel UpdatePropertyManagement(PimsProperty property);
 
-        IList<PimsPropPropActivity> GetManagementActivities(long propertyId);
+        IList<PimsPropertyActivity> GetActivities(long propertyId);
```

Comment:
I actually prefer ManagementActivity, but that is probably just personal preference and not something that needs changing.